src/main/java/sirius/biz/jobs/params/ParameterBuilder.java

```diff
@@ -261,7 +261,7 @@ protected V require(Map<String, String> context) {
      *
      * @return the name of the parameter
      */
-    protected String getName() {
```

Comment:
i think its better to share the name as a constant than sharing a mutable builder - as this is very easy to get wrong...(modify global state unintentionally)